projects/fusion-ui/components/layout/v4/components/layout-header/layout-header.component.html

```diff
@@ -1,7 +1,31 @@
 <div class="fu-header-top-row" [hidden]="!hasTopLine">
-    <div id="pageHeaderTopTeleportSlot"></div>
-    <div *ngFor="let teleportItem of teleportTopRowElements" [id]="teleportItem.id" class="fu-header-top-teleport-host"
-         [class.fu-flex-align-right]="teleportItem.isOnRight"></div>
+    <div id="pageHeaderTopTeleportSlot" [class.fu-has-skeleton]="teleportTopRowBaseSkeletons?.length" [style.gap]="teleportTopRowBaseSkeletonsGap">
+        @for (skeleton of teleportTopRowBaseSkeletons; track skeleton) {
+            <fusion-skeleton
+                    [shape]="skeleton.shape ?? 'rectangle'"
+                    [style.--fu-skeleton-width]="skeleton.width"
+                    [style.--fu-skeleton-height]="skeleton.height"
+                    [style.--fu-skeleton-border-radius]="skeleton.borderRadius"
+            ></fusion-skeleton>
+        }
+    </div>
+    @for (teleportItem of teleportTopRowElements; track teleportItem) {
+        <div [id]="teleportItem.id" class="fu-header-top-teleport-host"
+             [class.fu-flex-align-right]="teleportItem.isOnRight"
+             [class.fu-has-skeleton]="teleportItem?.skeletons?.length"
+             [style.gap]="teleportItem?.skeletonsGap"
+        >
+            @for (skeleton of teleportItem?.skeletons; track skeleton) {
+                <fusion-skeleton
+                        [shape]="skeleton.shape ?? 'rectangle'"
```

Comment:
Please take the default from const variable instead of hard coded value.

Reply:
Oh, here no need default. It takes default from component